html/pfappserver/root/src/views/Configuration/networks/interfaces/_components/TheForm.vue

```diff
@@ -71,6 +71,9 @@
           {{ $i18n.t(`Since NAT is disabled, PacketFence will adjust iptables to route traffic rather than using NAT. Make sure to add the routes on the system.`) }}
         </div>
       </b-form-group>
+      <form-group-nat-dns namespace="nat_dns"
+        :column-label="$i18n.t('Enable DNS NAT')"
```

Comment:
Add a help message describing what this is and why to use it since I'm not even sure myself